ManatiCR/manati-website
# Fix links url in our work page

## Issue
Some links had de wrong url in `/our-work.html`

Add the correct url to:
- Hipatía
- FDI
- DNN

## Steps to test 
 - Go to http://localhost:9000/our-work.html
- Ensure that the following items have the right url
- [x] Hipatía
- [x] FDI
- [x] DNN

<img width="950" alt="Screen Shot 2019-06-24 at 5 17 29 PM" src="https://user-images.githubusercontent.com/9342250/60058606-45206800-96a6-11e9-91fe-7d6d20af1f9d.png">
